src/guide_restli_client.md

```diff
@@ -1012,6 +1012,10 @@ This will set "Content-Type" header value as "application/json".
 
 This will set "Content-Type" header value as "application/x-pson"
 
+#### PROTOBUF2
```

Comment:
Are you capitalizing because this is part of an enum? If not, this doesn't need to be in all-caps

Reply:
yup, let's keep it capitalized.